gn3/api/llm.py

```diff
@@ -1,125 +1,113 @@
 """Api endpoints for gnqa"""
-from datetime import timedelta
 import json
-import sqlite3
-from redis import Redis
-
+from datetime import datetime, timezone
 from flask import Blueprint
 from flask import current_app
 from flask import jsonify
 from flask import request
 
 from gn3.llms.process import get_gnqa
-from gn3.llms.process import get_user_queries
-from gn3.llms.process import fetch_query_results
 from gn3.llms.errors import LLMError
 from gn3.auth.authorisation.oauth2.resource_server import require_oauth
 from gn3.auth import db
 
+
 gnqa = Blueprint("gnqa", __name__)
 
 
-@gnqa.route("/gnqna", methods=["POST"])
-def gnqna():
-    """Main gnqa endpoint"""
+@gnqa.route("/search", methods=["POST"])
```

Comment:
minor question: why do we use POST instead of GET or PUT? I see we store the results in a table, so perhaps `PUT` may be better because its expected to be idempotent.

[ref](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST)